### DISCUSSION TITLE
Welcome to gxi Discussions!

### DISCUSSION THREAD

**Discussion by aniketfuryrocks** [gxi-rs/gxi#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SamuraiCrow** [comment]
Greetings!  I am Samuel D. Crow, screen name SamuraiCrow, and I hope that when this project is completed it can be used in conjunction with Wasmer to make native internet apps so easy to write that they will obsolete the browser completely.  If it does the same for large, bulky operating systems that everyone currently uses, I won't complain.  :-)

**Reply by aniketfuryrocks** [reply]
Yeah, that's what I am aiming for. Let's hope for the best and we'll create the perfect overhead-free abstraction layer for cross-platform user interface development.

**Reply by Vince1996** [reply]
Hey Sam,
This is Vince, Kindly share your email address.

**Reply by SamuraiCrow** [reply]
My email is samueldcrow at gmail dot com .

**Reply by Vince1996** [reply]
Thanks,Sam.
Kindly check your email and give me a confirmation reply.

**Reply by SamuraiCrow** [reply]
Done!

**Reply by aniketfuryrocks** [reply]
What's going on @SamuraiCrow @Vince1996

**Reply by SamuraiCrow** [reply]
Sorry about the noise, @aniketfuryrocks .
